src/golang/lib/airflow/writer.go

```diff
@@ -37,23 +31,17 @@ func createWorkflowDagResult(
 		return nil, errors.New("Cannot create WorkflowDagResult for in progress Airflow DAG Run.")
 	}
 
-	workflowDagResult, err := workflowDagResultWriter.CreateWorkflowDagResult(ctx, dbDag.Id, db)
-	if err != nil {
-		return nil, err
-	}
-
-	changes := map[string]interface{}{
-		workflow_dag_result.CreatedAtColumn: *run.StartDate.Get(),
```

Comment:
We need the `CreatedAt` column to be in order to perform proper syncing. Se https://github.com/aqueducthq/aqueduct/blob/main/src/golang/lib/airflow/sync.go#L133

For now can you add an `updateWorkflowDagResult` call that sets the `CreatedAt` column.

Reply:
idk why this comment is missing in file view...I believe we set this field while creating the DAG result using the execState. Am I missing anything?